react/features/base/conference/middleware.js

```diff
@@ -81,6 +92,26 @@ function _connectionEstablished(store, next, action) {
 
 /**
  * Does extra sync up on properties that may need to be updated, after
+ * the conference failed or was left.
+ *
+ * @param {Store} store - The Redux store in which the specified action is being
+ * dispatched.
+ * @param {Dispatch} next - The Redux dispatch function to dispatch the
+ * specified action to the specified store.
+ * @param {Action} action - The Redux action {@link CONFERENCE_FAILED} or
+ * {@link CONFERENCE_LEFT} which is being dispatched in the specified store.
+ * @private
+ * @returns {Object} The new state that is the result of the reduction of the
+ * specified action.
+ */
+function _conferenceFailedOrLeft(store, next, action) {
+    store.dispatch(setAudioOnly(false));
```

Comment:
maybe something crashes here when leaving the conference and that's why the tests are failing ?